config/environments/development.rb

```diff
@@ -46,7 +46,7 @@
   config.assets.quiet = true
 
   # Raises error for missing translations.
-  config.i18n.raise_on_missing_translations = true
+  # config.i18n.raise_on_missing_translations = true
```

Comment:
What is the reason for commenting this?

Reply:
In theory this should not have caused any failing tests, because the CI runs this test https://github.com/alphagov/government-frontend/blob/bc185e1c4b450d11e1741c67de0ea21513c1ae5e/test/unit/locales_validation_test.rb#L4-L7

What were the errors?